tests/data/validation/enum/enum.go

```diff
@@ -368,6 +468,16 @@ var enumValues_EnumMyStringUntypedEnum = []interface{}{
 	"green",
 }
 
+// Verify checks all fields on the struct match the schema.
+func (plain EnumMyStringUntypedEnum) Verify() error {
+	for _, expected := range enumValues_EnumMyStringUntypedEnum {
+		if reflect.DeepEqual(v, expected) {
```

Comment:
where does this `v` come from? I don't see it declared anywhere, are you sure it's valid?